### ISSUE TITLE
508 Accessibility compliance

### ISSUE THREAD

**Issue by gerardlopez2001** [QMDevTeam/QMPDA#12]
For all user facing application features such as screens and menus, Section 508 Accessibility features should be enabled.

For more information, please see

google Drive:  QMDevTeam\508 Accessibility\Windows_Mobile_6_5_Professional.docx

also:
http://msdn.microsoft.com/en-us/library/aa932900.aspx

Related to QMDevTeam/QMGeneral#15

**Comment by gerardlopez2001** [comment]
Moved to Trello: https://trello.com/c/8V9FDus8
